dv/uvm/yaml/rtl_simulation.yaml

```diff
@@ -88,3 +88,22 @@
       -covbaserun test
     wave_opts: >
       -input <cwd>/ius.tcl
+
+- tool: riviera
+  env_var: ALDEC_PATH
+  compile:
+    cmd:
+      - "vlib <out>/work"
+      - "vlog -work <out>/work
+        +incdir+<ALDEC_PATH>/vlib/uvm-1.2/src
+        +incdir+${PRJ_DIR}/ibex/shared/rtl
+        -l uvm_1_2
+        -err VCP2694 W1
+        -f ibex_dv.f"
+
+  sim:
+    cmd: >
+      vsim -c <sim_opts> <cov_opts> -lib <out>/work +access +r+w -l <out>/logfile.log -do "run -all; endsim; quit -force" +ibex_tracer_file_base="trace_core"
```

Comment:
Is the plan to use `riviera_sim.tcl` here, or can the TCL file be removed?

Reply:
I've removed riviera_sim.tcl